CMakeLists.txt

```diff
@@ -301,6 +300,19 @@ if (BUILD_TESTING)
       PREFIX ""
       )
 
+   # Create two stubbed csfle libraries
+   add_library (stubbed-csfle SHARED test/csfle-stub.cpp)
+   add_library (stubbed-csfle-2 SHARED test/csfle-stub.cpp)
+
+   set_target_properties(stubbed-csfle stubbed-csfle-2 PROPERTIES
+      INCLUDE_DIRECTORIES "${CMAKE_CURRENT_SOURCE_DIR}/src"
+      LINK_LIBRARIES "mongo::bson_static"
```

Comment:
> Does that sound OK? If so, I can submit a PR to only build the `mongocrypt` target on 32 bit Windows.

Here is a [branch with the proposed changes](https://github.com/vector-of-bool/libmongocrypt/compare/use-csfle-lib...kevinAlbs:32bit-windows?expand=1).

On second thought, I disagree with my proposal. If PHPC builds libmongocrypt from source and runs on 32 bit Windows, that seems like reason to run tests of libmongocrypt.

Since this warning is only in the stubbed-csfle library, can the warning be ignored with pragmas?